namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlServiceConfig.scala

```diff
@@ -15,9 +22,13 @@ object HttpControlServiceConfig {
   val defaultAddr = new InetSocketAddress(InetAddress.getLoopbackAddress, 4180)
 }
 
-case class HttpControlServable(addr: InetSocketAddress, store: DtabStore) extends Servable {
+case class HttpControlServable(
+  addr: InetSocketAddress,
+  store: DtabStore,
+  namers: Ns => NameInterpreter
```

Comment:
tioli: naming-wise, it's a little odd that you call this `namers` in this constructor but `delegate` above in `HttpControlServiceConfig`, whereas `namers` is used for `Map[Path, Namer]` there. Not a huge deal, of course.